### PR TITLE
Add notice that components are outdated

### PR DESCRIPTION
**Pull request by creuter**
screen shot 2017-03-08 at 10 22 55 am

### CODE REVIEW

**Comment by kylefiedler** [comment]
Maybe make this more blatant? in the pink? put version numbers next to Bourbon and Neat?

File: source/_outdated_notice.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="refills-notice">
+  <span>
+    These components have not been updated to support the most recent versions
+    of Bourbon and Neat. We're working on another take of Refills with a focus
```
Comment (tysongach) [inline source/_outdated_notice.html.erb]: `take` → `iteration`?
`take` → `version`?

**Comment by creuter** [comment]
@kylefiedler thoughts?

screen shot 2017-03-03 at 4 23 11 pm

**Comment by creuter** [comment]
> Refills requires Bourbon 5.0+ and Neat <2.0, >=1.6. These have not been updated to support the most recent versions. We're working on another iteration with a focus on accessibility, which you can preview <here>.

**Comment by kylefiedler** [comment]
That's certainly more obvious

**Comment by creuter** [comment]
This is ready for re-review, please. Updated the PR description.

File: source/_outdated_notice.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="refills-wrapper">
+  <div class="refills-notice">
+    <span>
+      Refills requires Bourbon 5.0+ and Neat < 2.0, >= 1.6. These have not been
```
Comment (tysongach) [inline source/_outdated_notice.html.erb]: I think we can cut `These have not been updated to support the most recent versions.`.

**Pull request merged**